src/com/boxboat/jenkins/library/deploy/DeployTarget.groovy

```diff
@@ -0,0 +1,11 @@
+package com.boxboat.jenkins.library.deploy
+
+import com.boxboat.jenkins.library.config.BaseConfig
+
+class DeployTarget extends BaseConfig<Deployment> implements Serializable  {
+
+    String name
+
+    String key
```

Comment:
The library has been erring on the side of verbosity, so this should probably be `deployTargetKey`